go/router/templates/router/includes/actions.html

```diff
@@ -65,7 +65,7 @@
             Activating: Attempt to stop
         </button>
         <button
-         class="btn btn-danger action"
+         class="btn action"
          data-action="stop"
          data-url="{% router_screen router 'stop' %}"
          disabled>
```

Comment:
Sorry for not catching the inconsistencies in earlier reviews, but this change doesn't seem relevant to this PR, could we put irrelevant changes in their own PRs next time (and I'll promise to try do that too :)?